cypress/e2e/awx/resources/inventories.cy.ts

```diff
@@ -127,6 +127,21 @@ describe('inventories', () => {
     });
   });
 
+  it('test inventory group can be edited', () => {
+    cy.createInventoryHostGroup(organization).then((result) => {
+      const { inventory, group } = result;
+      cy.navigateTo('awx', 'inventories');
+      cy.clickTableRow(inventory.name);
+      cy.verifyPageTitle(inventory.name);
+      cy.clickLink(/^Groups$/);
+      cy.clickTableRowKebabAction(group.name as string, 'edit-group', true);
+      cy.verifyPageTitle('Edit group');
+      cy.get('[data-cy="name-form-group"]').type('-changed');
+      cy.get('[data-cy="Submit"]').click();
+      cy.verifyPageTitle(group.name + '-changed');
+    });
+  });
+
```

Comment:
Let's add another test case of editing from the detail page as well. This case handles editing from a table row.